Include/object.h

```diff
@@ -104,7 +104,14 @@ typedef struct _typeobject PyTypeObject;
  */
 typedef struct _object {
     _PyObject_HEAD_EXTRA
+#ifdef EXPERIMENTAL_ISOLATED_SUBINTERPRETERS
+    /* bpo-40533: Use an atomic variable for PyObject.ob_refcnt, to ensure that
+       Py_INCREF() and Py_DECREF() are safe when called in parallel, until
+       subinterpreters stop sharing Python objects. */
+    _Atomic Py_ssize_t ob_refcnt;
```

Comment:
Perhaps this is somewhat emphasized by the "experimental" part, but could we possibly make it a bit more crystal clear that this is only a temporary stopgap measure? I personally find that "until subinterpreters stop sharing Python objects" could be interpreted as a near indefinite amount of time; I'm not confident that most readers will go over the full context in the bpo issue and related discussions.

Normally, this wouldn't be a huge concern, but I think `object.h` is likely one of the most public-facing header files in the CPython, so it seems important to communicate this as clearly as possible.